apps/web/src/styles/7-components/_table.scss

```diff
@@ -0,0 +1,10 @@
+.govuk-accordion__section-content {
+  .govuk-summary-list:last-child {
```

Comment:
We use tabs for indentation. Would be great if you update all required files with it.

Not sure why editorconfig didn't picked it up for you.

Reply:
I think editorconfig is only applying to existing files, and if I create a new file, it's defaulting to my VScode preferences

Reply:
Hmm, it shouldn't, it should work for the entire workspace.

Reply:
I checked, and this is a tab. This is kinda the problem with tabs, in that they aren't predictable. If I re-apply prettier, it looks the same... Are you a tabs guy then? :(

Reply:
:) yeah

Tabs are more accessible than spaces because they represent indentation visually and non-visually. Tabs allow developers to choose the indentations that make the most sense for them. Tabs improve how code is interpreted in non-visual mediums.

also this:

> Just had a conversation about indentation with a blind programmer, and realized this is the most accessible indentation convention as well. Tabs as indentation means # tabs = indent level, and all spaces (including alignment spaces) can be simply ignored when reading out code.
— @fantasai (W3C CSSWG specifications writer, 2004 to present)